pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceAutoTopicCreationTest.java

```diff
@@ -526,4 +534,58 @@ public void testDynamicConfigurationTopicAutoCreationPartitionedWhenDefaultMoreT
         }
     }
 
+    @Test
+    public void testExtensibleLoadManagerImplInternalTopicAutoCreations()
+            throws PulsarAdminException, PulsarClientException {
+        pulsar.getConfiguration().setAllowAutoTopicCreation(true);
+        pulsar.getConfiguration().setAllowAutoTopicCreationType(TopicType.PARTITIONED);
+        pulsar.getConfiguration().setDefaultNumPartitions(3);
+        pulsar.getConfiguration().setMaxNumPartitionsPerPartitionedTopic(5);
+        final String namespaceName = NamespaceName.SYSTEM_NAMESPACE.toString();
+        TenantInfoImpl tenantInfo = new TenantInfoImpl();
+        tenantInfo.setAllowedClusters(Set.of(configClusterName));
+        admin.tenants().createTenant("pulsar", tenantInfo);
+        admin.namespaces().createNamespace(namespaceName);
+        admin.topics().createNonPartitionedTopic(ServiceUnitStateChannelImpl.TOPIC);
+        admin.topics().createNonPartitionedTopic(ExtensibleLoadManagerImpl.BROKER_LOAD_DATA_STORE_TOPIC);
+        admin.topics().createNonPartitionedTopic(ExtensibleLoadManagerImpl.TOP_BUNDLES_LOAD_DATA_STORE_TOPIC);
+
+        // clear the topics to test the auto creation of non-persistent topics.
+        ConcurrentOpenHashMap<String, CompletableFuture<Optional<Topic>>> topics =
+                pulsar.getBrokerService().getTopics();
+        ConcurrentOpenHashMap<String, CompletableFuture<Optional<Topic>>> oldTopics = new ConcurrentOpenHashMap<>();
+        topics.forEach((key, val) -> oldTopics.put(key, val));
+        topics.clear();
+
+        // The created persistent topic correctly can be found by
+        // pulsar.getPulsarResources().getTopicResources().persistentTopicExists(topic);
+        Producer producer = pulsarClient.newProducer().topic(ServiceUnitStateChannelImpl.TOPIC).create();
```

Comment:
Does this mean the topic `ServiceUnitStateChannelImpl.TOPIC" can be auto-created? Maybe we don't want it auto-create, right?

Reply:
No. We do pre-create `ServiceUnitStateChannelImpl.TOPIC`, and it won't be auto-created. As I commented, the producer creation can correctly find the existence of the "persistent" topic via `pulsar.getPulsarResources().getTopicResources().persistentTopicExists(topic);` check. The issue only happens for non-persisten topics now.

Reply:
It seems that it's because the method `MetaStoreImpl#getManagedLedgerInfo` can verify the path `/managed-ledgers/pulsar/system/persistent/loadbalancer-service-unit-state` exists, the topic can be created, and the producer can be created successfully, maybe we should also clean the cache ledgers of the `ManagedLedgerFactoryImpl`.

Reply:
>  maybe we should also clean the cache ledgers of the ManagedLedgerFactoryImpl.

sorry. I am not sure what you meant here. Is this related to this PR?

Reply:
I checked the logic, it seems that the reason why the producer can be created successfully is not "pulsar.getPulsarResources().getTopicResources().persistentTopicExists(topic)".

Reply:
Got it, sorry, I misunderstood.